idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -0,0 +1,1995 @@
+                            + self.total_variable_OM_cost[0] * self.capacity_factor
+                        )
+                        / (
+                            self.capacity_factor
+                            * self.net_power[0]
+                            * 8760
+                            * pyunits.hr
+                            / pyunits.year
+                        )
+                        + self.add_COE_cost
+                    )
+                )
+            if tonne_CO2_capture is not None:
+                if not hasattr(self, "tonne_CO2_capture"):
+                    self.tonne_CO2_capture = Param(
+                        initialize=tonne_CO2_capture, mutable=True, units=pyunits.ton
+                    )
+                self.cost_of_capture = Expression(
+                    expr=(
+                        (
+                            self.annualized_cost / pyunits.year
+                            + self.total_fixed_OM_cost / pyunits.year
+                            + self.total_variable_OM_cost[0] * self.capacity_factor
+                        )
+                        / self.tonne_CO2_capture
+                    )
+                )
+
+    @staticmethod
+    def initialize_build(self):
+        """
+        Here we can add intialization steps for the things we built in
+        build_process_costs.
+
+        Note that the aggregate costs will be initialized by the framework.
+        """
+        # TODO: For now,  no additional process level costs to initialize
+        pass
+
+    def report(self, export=False):
+
+        var_dict = {}
+
+        if hasattr(self, "total_TPC"):
+            var_dict["Total TPC [$MM]"] = value(self.total_TPC)
+
+        if hasattr(self, "total_overnight_capital"):
+            var_dict["Total Overnight Cost [$MM]"] = value(self.total_overnight_capital)
+
+        if hasattr(self, "total_as_spent_cost"):
+            var_dict["Total As Spent Cost [$MM]"] = value(self.total_as_spent_cost)
+
+        if hasattr(self, "annualized_cost"):
+            var_dict["Total Annualized Capital Cost [$MM/year]"] = value(
+                self.annualized_cost
+            )
+
+        if hasattr(self, "total_fixed_OM_cost"):
+            var_dict["Total fixed O&M cost [$MM/year]"] = value(
+                self.total_fixed_OM_cost
+            )
+
+        if hasattr(self, "total_variable_OM_cost"):
+            var_dict["Total variable O&M cost [$MM/year]"] = value(
+                self.total_variable_OM_cost[0]
+            )
+
+        if (
+            hasattr(self, "annualized_cost")
+            and hasattr(self, "total_fixed_OM_cost")
+            and hasattr(self, "total_variable_OM_cost")
+        ):
+            var_dict["Total Annualized Cost [$MM/year]"] = (
+                value(self.annualized_cost)
+                + value(self.total_fixed_OM_cost)
+                + self.capacity_factor * value(self.total_variable_OM_cost[0])
```

Comment:
Same comment in regards to multiplying the `total_variable_OM_cost` by the `capacity_factor`. Double check this